src/2018/transitioning/modules/path-clarity.md

```diff
@@ -11,22 +11,23 @@ features, but they end up *simplifying* the module system, to make it more
 clear as to what is going on.
 
 Note: During the 2018 edition preview, there are two variants of the module
-system under consideration, the "absolute use paths" variant and the "relative
+system under consideration, the "uniform paths" variant and the "anchored use
 paths" variant. Most of these changes apply to both variants; the two variant
 sections call out the differences between the two. We encourage testing of the
-new "relative paths" variant introduced in edition preview 2. The release of
+new "uniform paths" variant introduced in edition preview 2. The release of
 the 2018 edition will use one of these two variants.
```

Comment:
Perhaps:  "Once the Rust 2018 is released, only one of these two variants will be stabilized".